cli/root.go

```diff
@@ -67,8 +67,7 @@ const (
 	varOrganizationSelect = "organization"
 	varDisableDirect      = "disable-direct-connections"
 
-	notLoggedInMessage         = "You are not logged in. Try logging in using 'coder login <url>'."
-	notLoggedInURLSavedMessage = "You are not logged in. Try logging in using 'coder login'."
+	notLoggedInMessage = "You are not logged in. Try logging in using 'coder login <url>'."
```

Comment:
```suggestion
	notLoggedInMessage = "You are not logged in. Try logging in using 'coder login [url]'."
```
URL is sometimes optional, if we're only using on message then I'd prefer highlighting this.

Reply:
Well the funny thing is for you to get this message, you have to already hit the server.

Reply:
It seems like this shows the URL if you don't have it, and `coder login` if you do have it. That code wasn't really used properly!